docs/_includes/usage/cf-typography/usage.md

```diff
@@ -299,8 +298,28 @@ For more information, email
 </a>.
 ```
 
+### Block link
```

Comment:
Should this be `Block link modifier`?

Reply:
I suppose it technically could be, but it's never used in markup as a modifier class.

Reply:
What do you mean? The class is `a-link__block`

Reply:
As noted in the wording I'm adding in this PR, it's not currently ever used directly by writing it into the markup, and probably shouldn't be. It's mixed-in to jump link and link lists links.

Reply:
Ah gotcha, yeah that should be clearer in the actual example.